### PR TITLE
Reference to determine MongoDB.Driver version

### LINKED ISSUE #10715: Cannot install the MongoDB driver
I'm  running MongoDB version v3.4.2 
and 
dotnet version 2.2.103
when trying to install the MongoDB.Driver with 

`dotnet add BooksApi.csproj package MongoDB.Driver -v 2.7.2`

I get the following error:
```
info : Die PackageReference für das Paket "MongoDB.Driver" wird dem Projekt "BooksApi.csproj" hinzugefügt.
log  : Pakete für "C:\Users\alego\Desktop\BooksApi\BooksApi.csproj" werden wiederhergestellt...
error: "MongoDB.Driver (>= 2.7.3)" kann für ".NETCoreApp,Version=v2.2" nicht aufgelöst werden.
error: Das Paket "MongoDB.Driver" ist mit all Frameworks im Projekt "BooksApi.csproj" nicht kompatibel.
```
which basically means the MongoDB.Driver cannot be resolved for .NETCoreApp,Version=v2.2
and the package MongoDB.Driver is not compatible with all frameworks 

has anyone come across something similar?

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #10715

### CODE REVIEW

File: aspnetcore/tutorials/first-mongo-app.md [hunk 1]
```diff
@@ -139,10 +139,10 @@ The database is ready. You can start creating the ASP.NET Core web API.
 1. Go to **File** > **New** > **Project**.
 1. Select **ASP.NET Core Web Application**, name the project *BooksApi*, and click **OK**.
 1. Select the **.NET Core** target framework and **ASP.NET Core 2.1**. Select the **API** project template, and click **OK**:
-1. In the **Package Manager Console** window, navigate to the project root. Run the following command to install the .NET driver for MongoDB:
+1. Visit [.NET Driver for MongoDB: Releases](https://github.com/mongodb/mongo-csharp-driver/releases) to determine the latest stable version of the .NET driver for MongoDB. In the **Package Manager Console** window, navigate to the project root. Run the following command to install the .NET driver for MongoDB:
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-mongo-app.md]: Link to the NuGet page instead: https://www.nuget.org/packages/MongoDB.Driver/

File: aspnetcore/tutorials/first-mongo-app.md [hunk 2]
```diff
@@ -157,10 +157,10 @@ The database is ready. You can start creating the ASP.NET Core web API.
     A new ASP.NET Core web API project targeting .NET Core is generated and opened in Visual Studio Code.
 
 1. Click **Yes** when the *Required assets to build and debug are missing from 'BooksApi'. Add them?* notification appears.
-1. Open **Integrated Terminal** and navigate to the project root. Run the following command to install the .NET driver for MongoDB:
+1. Visit [.NET Driver for MongoDB: Releases](https://github.com/mongodb/mongo-csharp-driver/releases) to determine the latest stable version of the .NET driver for MongoDB. Open **Integrated Terminal** and navigate to the project root. Run the following command to install the .NET driver for MongoDB:
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-mongo-app.md]: Same suggestion as above

**Comment by guardrex** [comment]
Use "NuGet Gallery" ... sounds good. They do call it that at the site.

**Comment by Rick-Anderson** [comment]
Nice improvement.

**Pull request merged**